observation_portal/accounts/serializers.py

```diff
@@ -73,7 +73,8 @@ def get_proposals(self, obj):
                 proposal_details['time_used'] = obj.profile.time_used_in_proposal(proposal)
                 proposal_details['time_allocations'] = []
                 for time_allocation in proposal.timeallocation_set.filter(semester=proposal.current_semester):
-                    proposal_details['time_allocations'].append(time_allocation.as_dict(exclude=['semester', 'proposal', 'id']))
+                    proposal_details['time_allocations'].append(time_allocation.as_dict(
+                        exclude=['semester', 'proposal', 'id', 'instrument_type']))
```

Comment:
This is just a little fix to a previous change I made for getting proposal details from the profile endpoint - unrelated to the visibility bug.